.github/workflows/check.yml

```diff
@@ -51,5 +51,5 @@ jobs:
           RUSTFLAGS: -Awarnings
 
       - name: Run mdbook test
-        run: "ls -l && mdbook test --library-path ../alumet/target/debug,../alumet/target/debug/deps"
+        run: "ls -l && CARGO_PKG_VERSION=1.0 mdbook test --library-path ../alumet/target/debug,../alumet/target/debug/deps"
```

Comment:
what is this variable doing here?

Reply:
If I'm right, you won't need this "hack" anymore after moving the plugin source outside of the markdown file tree, as suggested by Titouan and I in another comment.

Reply:
should be ok now